src/main/java/com/epam/training/olha_haichenkova/task_3/page/YopmailMainPage.java

```diff
@@ -47,20 +48,11 @@ public YopmailClientPage checkInbox(){
     }
 
     private void closeAdvertisementIfAppear() {
+        wait.until(ExpectedConditions.not(ExpectedConditions.urlToBe(YOP_URL)));
```

Comment:
This waiter isn't related with the current method as the action causing the redirect is in other method